src/integration/java/io/pinecone/helpers/IndexManager.java

```diff
@@ -63,7 +64,7 @@ public static String createNewIndex(Pinecone pinecone, int dimension, String ind
         CreateIndexRequest createIndexRequest = new CreateIndexRequest()
                 .name(indexName)
                 .dimension(dimension)
-                .metric(IndexMetric.DOTPRODUCT)
+                .metric(IndexMetric.DOTPRODUCT) // Sparse-dense is only supported with DOTPRODUCT
```

Comment:
Added comment for future reference on why we create index with DOTPRODUCT metric.